script/deploy/tron/deploy-and-register-periphery.ts

```diff
@@ -802,11 +802,28 @@ async function deployAndRegisterPeripheryImpl(options: {
 
             const refundWalletHex = globalConfig.refundWallet
 
-            const constructorArgs = [wrappedNativeHex, refundWalletHex]
+            // Try to get converter address, default to zero address if not found
```

Comment:
same here, we can just pull from config instead of assigning a default value (since we have an entry for tron in the config)